config/services.php

```diff
@@ -36,5 +36,8 @@
 
     'twitter' => [
         'widget_id' => env('TWITTER_WIDGET_ID'),
+    ],
+    'sparkpost' => [
+        'secret' => env('SPARKPOST_SECRET')
```

Comment:
That's true, it is a general fix for using SparkPost. The TransportManager expects that to be there.

It's a bug that it is missing IMO, and thus is fixed here.